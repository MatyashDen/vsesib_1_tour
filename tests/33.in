30 6 2 1
15 2
