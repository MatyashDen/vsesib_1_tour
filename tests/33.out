3598
0RLE0LRE0DUE0UDE0RRLLE0RDULE0RUDLE0LLRRE0LDURE0LUDRE0DDUUE0UUDDE0RRRLLLE0RRDULLE0RRUDLLE0RDDUULE0RUUDDLE0LLDURRE0LLUDRRE0LDDUURE0LUUDDRE0DDDUUUE0UUUDDDE0RRRDULLLE0RRRUDLLLE0RRDDUULLE0RRUUDDLLE0RDDDUUULE0RUUUDDDLE0LLDDUURRE0LLUUDDRRE0LDDDUUURE0LUUUDDDRE0DDDDUUUUE0UUUUDDDDE0RRRDDUULLLE0RRRUUDDLLLE0RRDDDUUULLE0RRUUUDDDLLE0RDDDDUUUULE0RUUUUDDDDLE0LLDDDUUURRE0LLUUUDDDRRE0LDDDDUUUURE0LUUUUDDDDRE0DDDDDUUUUUE0UUUUUDDDDDE0RRRDDDUUULLLE0RRRUUUDDDLLLE0RRDDDDUUUULLE0RRUUUUDDDDLLE0RDDDDDUUUUULE0RUUUUUDDDDDLE0LLDDDDUUUURRE0LLUUUUDDDDRRE0LDDDDDUUUUURE0LUUUUUDDDDDRE0DDDDDDUUUUUUE0UUUUUUDDDDDDE0RRRDDDDUUUULLLE0RRRUUUUDDDDLLLE0RRDDDDDUUUUULLE0RRUUUUUDDDDDLLE0RDDDDDDUUUUUULE0RUUUUUUDDDDDDLE0LLDDDDDUUUUURRE0LLUUUUUDDDDDRRE0LDDDDDDUUUUUURE0LUUUUUUDDDDDDRE0DDDDDDDUUUUUUUE0UUUUUUUDDDDDDDE0RRRDDDDDUUUUULLLE0RRRUUUUUDDDDDLLLE0RRDDDDDDUUUUUULLE0RRUUUUUUDDDDDDLLE0RDDDDDDDUUUUUUULE0RUUUUUUUDDDDDDDLE0LLDDDDDDUUUUUURRE0LLUUUUUUDDDDDDRRE0LDDDDDDDUUUUUUURE0LUUUUUUUDDDDDDDRE0DDDDDDDDUUUUUUUUE0UUUUUUUUDDDDDDDDE0RRRDDDDDDUUUUUULLLE0RRRUUUUUUDDDDDDLLLE0RRDDDDDDDUUUUUUULLE0RRUUUUUUUDDDDDDDLLE0RDDDDDDDDUUUUUUUULE0RUUUUUUUUDDDDDDDDLE0LLDDDDDDDUUUUUUURRE0LLUUUUUUUDDDDDDDRRE0LDDDDDDDDUUUUUUUURE0LUUUUUUUUDDDDDDDDRE0DDDDDDDDDUUUUUUUUUE0UUUUUUUUUDDDDDDDDDE0RRRDDDDDDDUUUUUUULLLE0RRRUUUUUUUDDDDDDDLLLE0RRDDDDDDDDUUUUUUUULLE0RRUUUUUUUUDDDDDDDDLLE0RDDDDDDDDDUUUUUUUUULE0RUUUUUUUUUDDDDDDDDDLE0LLDDDDDDDDUUUUUUUURRE0LLUUUUUUUUDDDDDDDDRRE0LDDDDDDDDDUUUUUUUUURE0LUUUUUUUUUDDDDDDDDDRE0DDDDDDDDDDUUUUUUUUUUE0UUUUUUUUUUDDDDDDDDDDE0RRRDDDDDDDDUUUUUUUULLLE0RRRUUUUUUUUDDDDDDDDLLLE0RRDDDDDDDDDUUUUUUUUULLE0RRUUUUUUUUUDDDDDDDDDLLE0RDDDDDDDDDDUUUUUUUUUULE0RUUUUUUUUUUDDDDDDDDDDLE0LLDDDDDDDDDUUUUUUUUURRE0LLUUUUUUUUUDDDDDDDDDRRE0LDDDDDDDDDDUUUUUUUUUURE0LUUUUUUUUUUDDDDDDDDDDRE0DDDDDDDDDDDUUUUUUUUUUUE0UUUUUUUUUUUDDDDDDDDDDDE0RRRDDDDDDDDDUUUUUUUUULLLE0RRRUUUUUUUUUDDDDDDDDDLLLE0RRDDDDDDDDDDUUUUUUUUUULLE0RRUUUUUUUUUUDDDDDDDDDDLLE0RDDDDDDDDDDDUUUUUUUUUUULE0RUUUUUUUUUUUDDDDDDDDDDDLE0LLDDDDDDDDDDUUUUUUUUUURRE0LLUUUUUUUUUUDDDDDDDDDDRRE0LDDDDDDDDDDDUUUUUUUUUUURE0LUUUUUUUUUUUDDDDDDDDDDDRE0DDDDDDDDDDDDUUUUUUUUUUUUE0UUUUUUUUUUUUDDDDDDDDDDDDE0RRRDDDDDDDDDDUUUUUUUUUULLLE0RRRUUUUUUUUUUDDDDDDDDDDLLLE0RRDDDDDDDDDDDUUUUUUUUUUULLE0RRUUUUUUUUUUUDDDDDDDDDDDLLE0RDDDDDDDDDDDDUUUUUUUUUUUULE0RUUUUUUUUUUUUDDDDDDDDDDDDLE0LLDDDDDDDDDDDUUUUUUUUUUURRE0LLUUUUUUUUUUUDDDDDDDDDDDRRE0LDDDDDDDDDDDDUUUUUUUUUUUURE0LUUUUUUUUUUUUDDDDDDDDDDDDRE0DDDDDDDDDDDDDUUUUUUUUUUUUUE0UUUUUUUUUUUUUDDDDDDDDDDDDDE0RRRDDDDDDDDDDDUUUUUUUUUUULLLE0RRRUUUUUUUUUUUDDDDDDDDDDDLLLE0RRDDDDDDDDDDDDUUUUUUUUUUUULLE0RRUUUUUUUUUUUUDDDDDDDDDDDDLLE0RDDDDDDDDDDDDDUUUUUUUUUUUUULE0RUUUUUUUUUUUUUDDDDDDDDDDDDDLE0LLDDDDDDDDDDDDUUUUUUUUUUUURRE0LLUUUUUUUUUUUUDDDDDDDDDDDDRRE0LDDDDDDDDDDDDDUUUUUUUUUUUUURE0LUUUUUUUUUUUUUDDDDDDDDDDDDDRE0DDDDDDDDDDDDDDUUUUUUUUUUUUUUE0UUUUUUUUUUUUUUDDDDDDDDDDDDDDE0RRRDDDDDDDDDDDDUUUUUUUUUUUULLLE0RRRUUUUUUUUUUUUDDDDDDDDDDDDLLLE0RRDDDDDDDDDDDDDUUUUUUUUUUUUULLE0RRUUUUUUUUUUUUUDDDDDDDDDDDDDLLE0RDDDDDDDDDDDDDDUUUUUUUUUUUUUULE0RUUUUUUUUUUUUUUDDDDDDDDDDDDDDLE0LLDDDDDDDDDDDDDUUUUUUUUUUUUURRE0LLUUUUUUUUUUUUUDDDDDDDDDDDDDRRE0LDDDDDDDDDDDDDDUUUUUUUUUUUUUURE0LUUUUUUUUUUUUUUDDDDDDDDDDDDDDRE0UUUUUUUUUUUUUUUDDDDDDDDDDDDDDDE0RRRDDDDDDDDDDDDDUUUUUUUUUUUUULLLE0RRRUUUUUUUUUUUUUDDDDDDDDDDDDDLLLE0RRDDDDDDDDDDDDDDUUUUUUUUUUUUUULLE0RRUUUUUUUUUUUUUUDDDDDDDDDDDDDDLLE0RUUUUUUUUUUUUUUUDDDDDDDDDDDDDDDLE0LLDDDDDDDDDDDDDDUUUUUUUUUUUUUURRE0LLUUUUUUUUUUUUUUDDDDDDDDDDDDDDRRE0LUUUUUUUUUUUUUUUDDDDDDDDDDDDDDDRE0RRRDDDDDDDDDDDDDDUUUUUUUUUUUUUULLLE0RRRUUUUUUUUUUUUUUDDDDDDDDDDDDDDLLLE0RRUUUUUUUUUUUUUUUDDDDDDDDDDDDDDDLLE0LLUUUUUUUUUUUUUUUDDDDDDDDDDDDDDDRRE0RRRUUUUUUUUUUUUUUUDDDDDDDDDDDDDDDLLLE
WWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWWW
